ext/cl_khr_command_buffer.asciidoc

```diff
@@ -1539,6 +1546,173 @@ New errors:
   support {CL_COMMAND_BUFFER_CAPABILITY_DEVICE_SIDE_ENQUEUE_KHR} and _kernel_
   contains a kernel-enqueue call.
 
+
+The function
+
+include::{generated}/api/protos/clCommandSVMMemcpyKHR.txt[]
+
+Records a command to do an SVM memcpy operation.
+
+_command_buffer_ Refers to a valid command-buffer object.
+
+_command_queue_ Specifies the command-queue the command will be recorded to.
+Parameter is unused by this extension as only a single command-queue is
+supported and **must** be `NULL`.
+
+_dst_ptr_ is the pointer to a host (if the device supports system SVM)
+or SVM memory allocation where data is copied to.
+
+_src_ptr_ is the pointer to a host (if the device supports system SVM)
+or SVM memory allocation where data is copied from.
+
+_size_ is the size in bytes of data being copied.
+
+_sync_point_wait_list_, _num_sync_points_in_wait_list_ Specify
+synchronization-points that need to complete before this
+particular command can be executed.
+
+If _sync_point_wait_list_ is `NULL`, _num_sync_points_in_wait_list_
+must be 0. If _sync_point_wait_list_ is not `NULL`, the list of
+synchronization-points pointed to by _sync_point_wait_list_ must be
+valid and _num_sync_points_in_wait_list_ must be greater than 0.
+The synchronization-points specified in _sync_point_wait_list_ are
+*device-side* synchronization-points. The command-buffer associated
+with synchronization-points in _sync_point_wait_list_ must be the same
+as _command_buffer_. The memory associated with _sync_point_wait_list_
+can be reused or freed after the function returns.
+
+_sync_point_ Returns a synchronization-point ID that identifies this particular
+command. Synchronization-point objects are unique and can be used to
+identify this command later on. _sync_point_ can be `NULL` in which case it
+will not be possible for the application to record a wait for this command to
+complete. If the _sync_point_wait_list_ and the _sync_point_ arguments are not
+`NULL`, the _sync_point_ argument should not refer to an element of the
+_sync_point_wait_list_ array.
+
+_mutable_handle_ Returns a handle to the command. Handle is unused by
+this extension and must be passed as `NULL`.
+
+{clCommandSVMMemcpyKHR} returns {CL_SUCCESS} if the function is executed
+successfully. Otherwise, it returns the errors defined by
+{clEnqueueSVMMemcpy} except:
```

Comment:
Do we need to say anything about an OpenCL 3.0 device that does not support SVM here, or is this one of the errors that was inherited by `clEnqueueSVMMemcpy`?

For reference, I believe the relevant error is:

> [CL_INVALID_OPERATION](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#CL_INVALID_OPERATION) if the device associated with command queue does not support SVM.

Same comment also applies to the new SVM fill API.

Reply:
I believe we don't need to explicitly state it, but it should be inherited from the 3.0 spec error description.